cpu/fe310/include/sifive/platform.h

```diff
@@ -0,0 +1,129 @@
+// See LICENSE for license details.
+
+#ifndef _SIFIVE_PLATFORM_H
+#define _SIFIVE_PLATFORM_H
+
+// Some things missing from the official encoding.h
+#define MCAUSE_INT         0x80000000
+#define MCAUSE_CAUSE       0x7FFFFFFF
+
+
+
+#include "sifive/aon.h"
+#include "sifive/clint.h"
+#include "sifive/gpio.h"
+#include "sifive/otp.h"
+#include "sifive/plic.h"
+#include "sifive/prci.h"
+#include "sifive/pwm.h"
+#include "sifive/spi.h"
+#include "sifive/uart.h"
+
+
+/****************************************************************************
+ * Platform definitions
+ *****************************************************************************/
+
+// Memory map
+#define MASKROM_MEM_ADDR 		(0x00001000)
+#define TRAPVEC_TABLE_CTRL_ADDR	(0x00001010)
+#define OTP_MEM_ADDR 			(0x00020000)
+#define CLINT_CTRL_ADDR 		(0x02000000)
+#define PLIC_CTRL_ADDR 			(0x0C000000)
+#define AON_CTRL_ADDR 			(0x10000000)
+#define PRCI_CTRL_ADDR 			(0x10008000)
```

Comment:
Indention seems broken

Reply:
Vendor provided files.  It looks like renaming the sifive folder to vendor will suppress warnings